src/handlers/process-email/is-valid-incoming-message-body.ts

```diff
@@ -0,0 +1,5 @@
+export const isValidIncomingMessageBody = (
+  maybeIncomingMessageBody: unknown
+): maybeIncomingMessageBody is string =>
+  typeof maybeIncomingMessageBody === "string" &&
+  maybeIncomingMessageBody.includes("MIME-Version");
```

Comment:
I believe we only work with emails that have attachments (pdf or csv) so if an email arrives without one, that is invalid and it should fail early in the process.
Also this typeguard is maybe just temporarily in here as, once the whole flow is hooked up and we're receiving an encrypted email, this will need to change?

Reply:
I will add a comment saying this will need attention as part of BTM-575, but if you have any other suggestions let me know.

Reply:
I will add a comment saying this will need attention as part of BTM-575, but if you have any other suggestions let me know.

Reply:
All done.